src/pages/ProductList/components/ChangeBtn/ChangeBtn.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import './ChangeBtn.scss';
 
-const ChangeBtn = () => {
+const ChangeBtn = ({ setSearchParams, offset, searchParams }) => {
+  const movePage = pageNumber => {
+    searchParams.set('offset', (pageNumber - 1) * 3);
```

Comment:
```suggestion
    searchParams.set('offset', (pageNumber - 1) * 3);
```
여기서 3이 의미하는거랑, 1이 의미하는게 숫자로만 봐선 예측하기 어렵습니다.
변하지않는 상수라면, 변수에 담아서 의미를 알 수 있게 해주시는것도 방법입니다.

Reply:
수정했습니다.